erpnext/accounts/doctype/sales_invoice/test_sales_invoice.py

```
@@ -392,6 +392,130 @@ def test_sales_invoice_with_discount_and_inclusive_tax(self):
+			"taxes",
+			{
+				"charge_type": "On Net Total",
+				"account_head": "_Test Account Service Tax - _TC",
+				"cost_center": "_Test Cost Center - _TC",
+				"description": "Service Tax",
+				"rate": 14,
+			},
+		)
+		si.insert()
+
+		# Item 1: 3 × 200 = 600, Item 2: 2 × 350 = 700, Total = 1300
```

Ruff (0.15.0)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)